### ISSUE TITLE
Westmere EP X5690

### ISSUE THREAD

**Issue by startergo** [Piker-Alpha/ssdtPRGen.sh#496]
Please add support for Westmere EP X5690:
X5690,130,1600,3460,3730,6,12,4,133

**Comment by h9826790** [comment]
Yeah, please. I am not sure if the same code will work for W3690 as well.

https://ark.intel.com/content/www/us/en/ark/products/52586/intel-xeon-processor-w3690-12m-cache-3-46-ghz-6-40-gt-s-intel-qpi.html

They have identical spec. W3690,130,640,3460,3730,6,12.

If two separated entries are required, can you also add W3690's support into the data base please?

**Comment by startergo** [comment]
> Yeah, please. I am not sure if the same code will work for W3690 as well.
> 
> https://ark.intel.com/content/www/us/en/ark/products/52586/intel-xeon-processor-w3690-12m-cache-3-46-ghz-6-40-gt-s-intel-qpi.html
> 
> They have identical spec. W3690,130,640,3460,3730,6,12.
> 
> If two separated entries are required, can you also add W3690's support into the data base please?

It looks like in the example the layout is different:
More info here:
https://github.com/Piker-Alpha/ssdtPRGen.sh/issues/284
https://github.com/Piker-Alpha/ssdtPRGen.sh/issues/288
https://github.com/Piker-Alpha/ssdtPRGen.sh/issues/231
https://github.com/Piker-Alpha/ssdtPRGen.sh/issues/236
